src/runtime_src/core/edge/sw_em/generic_pcie_hal2/shim.cxx

```diff
@@ -24,11 +24,13 @@
 #include "core/common/xclbin_parser.h"
 #include <errno.h>
 #include <unistd.h>
+#include <boost/lexical_cast.hpp>
```

Comment:
boost/lexical_cast has long since been replaced by std::to_string.  Please eliminate the use of boost here.